### ISSUE TITLE
Non-Code: Add extension to packagist

### ISSUE THREAD

**Issue by jpmschuler** [franzholz/static_info_tables_de#5]
TL;DR: Thanks for maintaining this - took me a while to find though. Please improve the discoverability with packagist and ease the installation for composer.

https://github.com/franzholz/static_info_tables_de.git should be added as jambagecom/static-info-tables-de on [packagist](https://packagist.org). The setup ist done in minutes. If github is used like here and the hooks/integration is done like the packagist setup recommends, it is a one-time-setup, with no additional steps ever as everything is updated via post-commit-hooks (or something similar).

> GitHub Service Hook
> Enabling the Packagist service hook ensures that your package will always be updated instantly when you push to GitHub.
> To do so you can:
> Go to your GitHub repository
> Click the "Settings" button
> Click "Integrations & services"
> Add a "Packagist" service, and configure it with your API token, plus your Packagist username
> Check the "Active" box and submit the form
> You can then hit the "Test Service" button to trigger it and check if Packagist removes the warning about the package not being auto-updated.`

https://insight.helhum.io/post/174851830180/improvements-to-composertypo3org adds that in extensions.typo3.org you should add the packagist link afterwards, although that would need to take over the extension in TER, which is a different topic.

**Comment by franzholz** [comment]
However Github shows this warning:

    

> Services
> 
> Services are pre-built integrations that perform certain actions when events occur on GitHub.
> Note: GitHub Services have been deprecated. Please contact your integrator for more information on how to migrate or replace a service with webhooks or GitHub Apps. 

It seems not to be possible to get the extension key back.

**Comment by jpmschuler** [comment]
Mea culpa, pasted old info: It is now "GitHub Hook for Packagist"
https://packagist.org/about#how-to-update-packages

**Comment by franzholz** [comment]
The webhook has already been entered before.


https://packagist.org/packages/jambagecom/static-info-tables-de

shows no conflicts

**Comment by jpmschuler** [comment]
then i'm double stupid to find the package, sry for the disturbance.
